### PR TITLE
fix: merge migrations and fixed a few casings

### PR DESCRIPTION
**Pull request by markusahlstrand**
Added all the migrations to the init one which makes it easier to read.

Looks like we still have some tenantId casings.

### CODE REVIEW

File: src/migrate/migrations/2022-12-11T09:17:35_init.ts [hunk 1]
```diff
@@ -96,9 +106,25 @@ export async function up(db: Kysely<Database>): Promise<void> {
...
+    .addColumn("email_api_key", "varchar(255)")
+    .addColumn("dkim_private_key", "varchar(2048)")
+    .addColumn("dkim_public_key", "varchar(2048)")
+    .addColumn("created_at", "varchar(255)")
+    .addColumn("modified_at", "varchar(255)")
+    .execute();
 }
 
 export async function down(db: Kysely<Database>): Promise<void> {
+  await db.schema.dropTable("domains").execute();
```
Comment (danielgent) [inline src/migrate/migrations/2022-12-11T09:17:35_init.ts:127]: nice! I didn't go through all the migration scripts clearly :smile: 

I was getting snow blind after all those changes yesterday :upside_down_face:

File: src/routes/tsoa/connections.ts [hunk 2]
```diff
@@ -159,7 +159,7 @@ export class ConnectionsController extends Controller {
   @Security("oauth2managementApi", [""])
   public async putConnection(
     @Request() request: RequestWithContext,
-    @Path("tenantId") tenantId: string,
+    @Path("tenantId") tenant_id: string,
```
Comment (danielgent) [inline src/routes/tsoa/connections.ts:162]: ah ok yes. this makes sense. no reason to maintain camelCase here :smile:

File: src/routes/tsoa/connections.ts [hunk 3]
```diff
@@ -38,7 +38,7 @@ export class ConnectionsController extends Controller {
     const db = getDb(ctx.env);
     const query = db
       .selectFrom("connections")
-      .where("connections.tenantId", "=", tenantId);
+      .where("connections.tenant_id", "=", tenantId);
```
Comment (danielgent) [inline src/routes/tsoa/connections.ts:41]: cool ok. so the types were failing here right? :+1:   I imagined I'd have missed something

File: src/migrate/migrations/index.ts [hunk 4]
```diff
@@ -1,16 +1,6 @@
 import * as m1_init from "./2022-12-11T09:17:35_init";
-import * as m2_domains from "./2023-08-15T09:17:35_domains";
-import * as m3_connection_scopes from "./2023-08-18T14:41:03_connection_scope";
-import * as m4_dkim from "./2023-08-22T15:06:26_dkim";
-import * as m5_styles from "./2023-08-28T09:12:44_styles";
-import * as m6_mailgun from "./2023-08-31T13:50:12_mailgun";
```
Comment (danielgent) [inline src/migrate/migrations/index.ts:6]: *to me* it looks like you've moved the `domains` script into the `init` one, but then the others are gone

*can you confirm that is correct?* @markusahlstrand  :smile:

**Review by danielgent — APPROVED** [review]
:checkered_flag:

File: src/migrate/migrations/index.ts (hunk 4 above)
Comment (markusahlstrand) [reply src/migrate/migrations/index.ts:6]: The other were just adding more columns

**Pull request merged**